addons/binding/org.openhab.binding.astro/ESH-INF/config/config.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<config-description:config-descriptions
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:config-description="http://eclipse.org/smarthome/schemas/config-description/v1.0.0"
+    xsi:schemaLocation="http://eclipse.org/smarthome/schemas/config-description/v1.0.0
+        http://eclipse.org/smarthome/schemas/config-description-1.0.0.xsd">
+
+  <config-description uri="config:astro-config">
```

Comment:
This uri needs to follow a certain naming convention in order to make sure that there are no clashes, see https://github.com/eclipse/smarthome/blob/master/docs/sources/architecture/configuration.md#xml-structure-for-configuration-descriptions. It probably should be ```thing-type://astro:config```.